app/controllers/ToDo/DestroyController.js

```diff
@@ -16,15 +16,20 @@ class DestroyController {
      *   @apiError Forbidden    ToDo element belongs to other User
      *   @apiError BadRequest
      */
-    constructor(todoRepository) {
+    constructor(todoRepository, isValidUuidServices) {
         this.todoRepository = todoRepository;
+        this.isValidUuidServices = isValidUuidServices;
     }
 
     async invoke(request, response) {
         const {
             params: { id: todoId }
         } = request;
 
+        if (!(await this.isValidUuidServices.handle(todoId))) {
```

Comment:
await not needed

Reply:
also call it `isUUIDValidHandler` everywhere

Reply:
Wait a minute. Checking params should be done in validators. 
Please not create services with method with one simple line. 
In validators it's one line: `validate(uuid);`
You created service: 6 lines, injected it several times, multiple changes. 

Validators, validators, validators!!!!!